src/store/oauth/types.ts

```diff
@@ -24,6 +24,7 @@ export interface TwitchToken {
 }
 
 export enum BnetOAuthRegion {
+  us = "us",
```

Comment:
We will have to check if this has any downstream implications if the session is established via another gateway now.

Reply:
I think it's certainly less risky to leave this as-is and call it global, we've already effectively established that the Europe gateway is what we use globally.